blocks/editable/index.js

```diff
@@ -705,7 +705,7 @@ export default class Editable extends Component {
 		}
 
 		content = renderToString( content );
-		this.editor.setContent( content, { format: 'raw' } );
+		this.editor.setContent( content );
```

Comment:
Could this have other consequences? cc @iseulde

Reply:
Previous discussion: https://github.com/WordPress/gutenberg/issues/4205#issuecomment-355169202

Reply:
Another previous discussion: https://github.com/WordPress/gutenberg/pull/667#discussion_r114950994

Reply:
I don't really see a problem with this if we want the content to be validated. With the `raw` setting, we assume the content is valid and exactly as it needs to be set in the `contentEditable` field. I think it's a bit weird though to be getting the raw content (through parsing the nodes) and setting the content with filters and validation.

Reply:
In other words, without the user changing anything about the block, the block attributes will have changed?

Reply:
Why does this not happen with a normal list block? Why only a reusable list block? Can it not behave in the same way?

Reply:
The bug occurs with normal list blocks too—it's just harder to come across. For example:

1. Create a new post
2. Create a list block with a few items
3. Save the post as a draft
4. Refresh the page
5. Add a new item to the list block
6. Click *Undo*